examples/browser/README.md

```diff
@@ -0,0 +1,19 @@
+Jupyter JS Services Browser Example
+===================================
+
+This example demonstrates using Jupyter JS Services from the browser using
+Webpack. The python script `main.py` is used to start a Jupyter Notebook Server
+and serve the Webpack bundle.  The example is written in TypeScript using ES6
```

Comment:
It might be helpful to highlight that this example uses ES6 and the Node example uses ES5. Perhaps a note at the top or bolding the sentences that mention ES6 and ES5.